FWCore/Skeletons/mkTemplates/EDAnalyzer/EDAnalyzer.cc

```diff
@@ -135,8 +135,8 @@ void __class__::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
 
   //Specify that only 'tracks' is allowed
   //To use, remove the default given above and uncomment below
-  //ParameterSetDescription desc;
-  //desc.addUntracked<edm::InputTag>("tracks","ctfWithMaterialTracks");
+  //edm::ParameterSetDescription desc;
+  //desc.addUntracked<edm::InputTag>("tracks", edm::InputTag("ctfWithMaterialTracks"));
   //descriptions.addWithDefaultLabel(desc);
```

Comment:
@Dr15Jones Beyond this PR, but I started to wonder if it would make sense to make this "actual implementation" the default? In case people don't want to implement `fillDescriptions()` properly, I think I'd rather see the the function not being implemented than the explicit `desc.setUnknown()`.